solutions/kubernetes/cortx-deploy.sh

```diff
@@ -84,19 +84,26 @@ function setup_cluster {
        WORKER_NODES=$(cat "$HOST_FILE" | grep -v "$MASTER_NODE" | awk -F[,] '{print $1}' | cut -d'=' -f2)
     fi
 
-    # Setup all worker nodes in parallel in case of multinode cluster.
+    # Setup master node and all worker nodes in parallel in case of multinode cluster.
     if [ "$(wc -l < $HOST_FILE)" -ne "1" ]; then
+       NODES=$(wc -l < $HOST_FILE)
+       TAINTED_NODES=$(ssh -o 'StrictHostKeyChecking=no' ssc-vm-rhev4-1222.colo.seagate.com bash << EOF
```

Comment:
can we remove hardcoded VM value in command?

Reply:
Oh...let me fix that.

Reply:
Done.